mission_yaml/SEA45_M48.yml

```diff
@@ -1,5 +1,5 @@
 metadata:
-  acknowledgement: This study used data collected and made freely available by Voice of the Ocean Foundation 
+  acknowledgement: This study used data collected and made freely available by Voice of the Ocean Foundation
```

Comment:
I don't remember changing anything in this line. Might be my silly editor. But please change line 33
contributor_role: Data Processor, PI, Glider Technician, Glider Technician, Glider Technician

Reply:
Yeah, looks like the editor making changes. If it continues causing problems we might set up some kind of auto-code formatter

Reply:
You can change that line in this PR i f you want to. May as well do them all al once